www_src/lib/keyboard.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  simulateTabPress: function () {
+    var current = document.activeElement;
+    var elements = document.getElementsByTagName('input');
+    var idx = parseInt(current.getAttribute('tabIndex'), 10) + 1;
```

Comment:
Why would I use a coercion here when it is harder to read + throws a lint error?